.eslintrc.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
We can keep all those configs files inside the web folder and focus on the web project right now. Actually it was my bad not ask if would be okay create a mono repo project to simplify as we will need a simple FE and BE projects.
What do you say?

Reply:
A mono repo is new for me, it would be perfect for discussing at on next meeting! But sounds like something we should aim for after a quick google search.